### ISSUE TITLE
Can I execute tests usin dojo 'require' instead 'RequireJS' library

### ISSUE THREAD

**Issue by csanchezr** [mmanela/chutzpah#334]
Can I use `dojo` (and its require feature) framework with `chutzpah` plugin? I've seen that in the `chutzpah.json` configuration file we can define the path of the require plugin and its configuration. But I tried to configure dojo without success...

My folder structure looks like this:

	Project
		js
			GUIFramework
				external
					dojo
						dojo-1.9.js
					_main.js
				WaspFramework
					Core
						Utils
							WaspDateTime.js
							WaspUtils.js							
					
		UnitTests
			Custom
				test
					WaspDateTimeTest.js
				chutzpah.json

The content of the `chutzpah.json` setup file:
	

    {
    	"Framework": "qunit",
    	"TestHarnessReferenceMode": "AMD",
    	"TestHarnessLocationMode": "SettingsFileAdjacent",
    	"AMDBasePath": "../../js/GUIFramework",
    	"References": [
    		{ "Path": "../../js/GUIFramework/external/_main.js" }, 
    		{"Path" : "../../js/GUIFramework/external/dojo/dojo-1.9.js" }
    	],
    	"Tests" : [
    	  {"Path": "tests"}
    	]
     }

`WaspDateTimeTest.js`file content:
	

    require(['WaspUtils/WaspDateTime'], function (waspDateTime) {
    		module( 'WaspDateTime test')
    		test( 'getSecondsFromIso8601 function', function( assert ) {
    		  var value = 35000;
    		  var result = WaspDateTime.getSecondsFromIso8601('PT9H43M20S');
    		  equal( value, result, 'We expect value "PT9H43M20S" to be 35000 seconds');
    		});
    	});
	
Content of `_main.js` file summarized. I've only included most relevant entries:

    dojoConfig = {
        async: true,
        baseUrl: '../GUIFramework/',
        tlmSiblingOfDojo: false,
        paths: {
            dojo: 'external/dojo/dojo-1.9',
    
            jQuery: 'external/jquery/jquery-1.9.1',
            jQueryUI: 'external/jquery/jquery-ui.min',
    
            use: 'external/use/use.min',
            WaspFramework: 'WaspFramework'
    
        },
        packages: [
            { name: 'WaspUtils', location: 'WaspFramework/Core/Utils', main: 'WaspUtils' }     
        ],
    
        use: {
            'jQuery': {
                attach: '$'
            },
            'jQueryUI': {
                deps: ['use!jQuery'],
                attach: function (jQuery) {
                    require.includeFile('externalCss/jquery/jquery-ui-1.8.19/themes/smoothness/jquery-ui-1.8.19.custom.css');
                }
            }
        }
    };

I can't see any error between the folder structure and the `chutzpah.json`config file. Perhaps I'm wrong. I'm also frustrated because running the `QUnit` tests in the browser, with same configuration files and tests, are being executed successfully. I know that this issue is a lot different from the browser runner, but I hope to be able to run it.

I've even installed the 'Test explorer' plugin for `chutzpah` and my tests are not being displayed on it. On the other hand, I've executed a little test with `chutzpah` samples with `RequireJS`, and they are working fine, and its tests cases are displayed on the 'Test explorer'

Obviously, I need to use `dojo` because I use it for my classes' structure so...I need it. Actually my final goal is to use `QUnit` with `Sinon.JS`, but I hope that when I solve the issue with `dojo` 
I won't have more problems (it is not a realistic dream, I know).

Basically, this issue is almost the same text as the [`stackoverflow` post](http://stackoverflow.com/questions/28905394/is-chutzpah-compatible-with-dojo-require-and-sinon) where I explained my problem too. 

If you need some other information or you want know anything else about the issue don't hesitate to ask me. I could even provide you a project to test it if I knew how to upload it.

**Comment by mmanela** [comment]
Can you create a git repo demonstrating your set up that I can clone and run?

**Comment by csanchezr** [comment]
Hi @mmanela,

I have been moved from this task and the responsible of that have implemented his own solution, so I think we should no longer care about it.

So sorry to bother you and thank you for your time.
